src/promptflow-core/promptflow/core/_model_configuration.py

```diff
@@ -36,9 +53,18 @@ class OpenAIModelConfiguration(ModelConfiguration):
 
     def __post_init__(self):
         self._type = ConnectionType.OPEN_AI
-        if any([self.base_url, self.api_key, self.api_version, self.organization]) and self.connection:
+        if any([self.base_url, self.api_key, self.organization]) and self.connection:
             raise InvalidConnectionError("Cannot configure model config and connection at the same time.")
 
+    @classmethod
+    def from_connection(cls, connection: OpenAIConnection, model: str, organization: str = None, **kwargs):
```

Comment:
remove **kwargs if not used.

Reply:
This was for backwards compatibility, removed as offline synced.